src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -1,378 +1,420 @@
-                            .getUrlForPath((String) data[1]),
-                    (String) data[7]);
-            Assert.assertEquals(c.newInstance((String) data[0], "dev-dmytror", (Boolean) data[2], (Boolean) data[3], (String) data[10])
-                            .getUrlForPath((String) data[1]),
-                    (String) data[8]);
-            Assert.assertEquals(c.newInstance((String) data[0], "staging", (Boolean) data[2], (Boolean) data[3], (String) data[10])
-                            .getUrlForPath((String) data[1]),
-                    (String) data[9]);
-        }
+    for (Object[] data : TEST_DATA_LANGUAGE) {
+      Assert.assertEquals(
+          c.newInstance((String) data[0], "prod", (Boolean) data[2], (Boolean) data[3],
+              (String) data[10])
+              .getUrlForPath((String) data[1]), (String) data[4]);
+      Assert.assertEquals(
+          c.newInstance((String) data[0], "preview", (Boolean) data[2], (Boolean) data[3],
+              (String) data[10])
+              .getUrlForPath((String) data[1]),
+          (String) data[5]);
+      Assert.assertEquals(
+          c.newInstance((String) data[0], "sandbox", (Boolean) data[2], (Boolean) data[3],
+              (String) data[10])
+              .getUrlForPath((String) data[1]),
+          (String) data[6]);
+      Assert.assertEquals(c.newInstance((String) data[0], "sandbox-mercurydev", (Boolean) data[2],
+          (Boolean) data[3], (String) data[10])
+              .getUrlForPath((String) data[1]),
+          (String) data[7]);
+      Assert.assertEquals(
+          c.newInstance((String) data[0], "dev-dmytror", (Boolean) data[2], (Boolean) data[3],
+              (String) data[10])
+              .getUrlForPath((String) data[1]),
+          (String) data[8]);
+      Assert.assertEquals(
+          c.newInstance((String) data[0], "staging", (Boolean) data[2], (Boolean) data[3],
+              (String) data[10])
+              .getUrlForPath((String) data[1]),
+          (String) data[9]);
     }
+  }
 
-    @Test(groups = {"TestUrlBuilder", "unitTests"})
-    public void getWikiaUrlTests() throws Exception{
-        Constructor<UrlBuilder> c = getUrlBuilderConstructor();
+  @Test(groups = {"TestUrlBuilder", "unitTests"})
+  public void getWikiaUrlTests() throws Exception {
+    Constructor<UrlBuilder> c = getUrlBuilderConstructor();
 
-        Assert.assertEquals(c.newInstance("mediawiki119","preview", false, false, DEFAULT_LANGUAGE).getWikiGlobalURL(),
-                "http://www.preview.wikia.com");
-        Assert.assertEquals(c.newInstance("mediawiki119","sandbox-s1", false, false, DEFAULT_LANGUAGE).getWikiGlobalURL(),
-                "http://www.sandbox-s1.wikia.com");
-        Assert.assertEquals(c.newInstance("mediawiki119","prod", false, false, DEFAULT_LANGUAGE).getWikiGlobalURL(),
-                "http://www.wikia.com");
-    }
+    Assert.assertEquals(c.newInstance("mediawiki119", "preview", false, false, DEFAULT_LANGUAGE)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "mediawiki119" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)